.github/dependabot.yaml

```diff
@@ -0,0 +1,35 @@
+version: 2
```

Comment:
Commit message: remember to include the scope.

Reply:
Good call. Unfortunately, I'm not sure what the scope should be.  What do you think is appropriate?

Reply:
"Dependabot" seems solid.

Reply:
Thank you!